### ISSUE TITLE
check_subrc: find ... match count

### ISSUE THREAD

**Issue by mbtools** [abaplint/abaplint#1632]
`find` with `match count` should not trigger this error since the match count will be evaluated separately. One would check for count = 0, not sy-subrc <> 0.

```abap
REPORT zfoobar.
FIND ALL OCCURRENCES OF 'a' IN 'abc' MATCH COUNT DATA(i).
```

/zfoobar.prog.abap[2, 1]: Check sy-subrc(check_subrc)

PS: I would skip the error also for `find` with `match offset/length`, `results`, or `submatches` since you would typically check the filled variables rather than sy-subrc.

**Comment by larshp** [comment]
- FIND with MATCH COUNT
- FIND with MATCH LENGTH
- FIND with RESULTS
- FIND with SUBMATCHES

anything else from https://syntax.abaplint.org/?filter=find#/statement/Find ?

**Comment by mbtools** [comment]
- FIND with MATCH OFFSET
- FIND with MATCH LINE

It's all options which have a `Target`

**Comment by larshp** [comment]
try again, 2.63.15

**Comment by mbtools** [comment]
👍
